.travis.yml

```diff
@@ -7,8 +7,7 @@ env:
         - PYTHONPATH=$PWD:$PYTHONPATH
 
 install:
-    # Install PyTorch wheel using the install script
-    - bash scripts/install_pytorch.sh
+    - pip install torch
```

Comment:
this will only work for OSX right? otherwise you need the URL of the whl `http://download.pytorch.org/whl/cpu/torch-0.4.0-cp35-cp35m-linux_x86_64.whl `

Reply:
I think you are right, from what I remember of yesteryears. Will change.